simulator/resource/default_cfg.yml

```diff
@@ -0,0 +1,31 @@
+# Restart proposing, if decision wasn't made in given time.
```

Comment:
@stepanvalyavskiy since we're using yaml for config, I'd suggest being more yamly: group related properties(e.g. `frontend` or `backend`), use group names as enable option (e.g. `dropRate`). This config could be refactored into something like this:
```yaml
frontend:
  proposerTimeout: 300
  replicas: 1
  crashRate: 0.1
backend:
  replicas: 3
  crashRate: 0
network:
  dropRate: 0.1
  async: true
  delay: 0.5
```